shared/src/test/java/TestStatsProcessor.java

```diff
@@ -57,4 +57,18 @@ public void getLeaderboard() {
         assertThat(leaderboard[1].getPlayerName(), equalTo("CHS"));
         assertThat(leaderboard[1].getHighestScore(), equalTo(18));
     }
+
+    @Test
+    public void getEmptyAllStats() {
+        Database db = Database.create("concrete", "dummyFilename");
```

Comment:
while you're at it can you change the `ichiba.csv` one as well. I don't think I intended to commit that...